apis/sgv2-graphqlapi/src/main/resources/application.yaml

```diff
@@ -48,6 +59,19 @@ quarkus:
     dev-mode:
       force-server-start: false
 
+  # HTTP settings
+  http:
+    port: ${STARGATE_GRAPHQL_PORT:8080}
```

Comment:
Similar to above, maybe just remove and rely on Quarkus' overrides (I don't think this is used anywhere, docker-compose was changed not to use it)

Reply:
Removing.